src/libnnstreamer-edge/nnstreamer-edge-mqtt.c

```diff
@@ -496,7 +497,7 @@ nns_edge_mqtt_get_message (nns_edge_h edge_h, char **msg)
 
   eh = (nns_edge_handle_s *) edge_h;
 
-  if (!NNS_EDGE_MAGIC_IS_VALID (eh) || !eh->broker_h) {
+  if (!NNS_EDGE_MAGIC_IS_VALID (eh) || !eh->broker_h || !msg) {
```

Comment:
can you separate this condition (!msg) to print exact error message?

Reply:
Updated!